install.ps1

```diff
@@ -341,4 +341,177 @@ function Install-Mondoo {
   # reset erroractionpreference
   $erroractionpreference = $previous_erroractionpreference
   }
-}
\ No newline at end of file
+}
+
+# SIG # Begin signature block
```

Comment:
Since the signature should be added by the github action, I would like to keep it out of the PR